nservicebus/outbox/index.md

```diff
@@ -47,21 +47,26 @@ Returning to the earlier example of a message handler which creates a `User` and
 
 ```mermaid
 graph TD
+  subgraph Phase 1
   receiveMessage(1. Receive incoming message)
   createTx(2. Begin transaction)
   dedupe{3. Deduplication}
-  handler(4. Execute handler)
+  messageHandler(4. Execute message handler)
   storeOutbox(5. Store outgoing<br/>messages in outbox)
   commitTx(6. Commit transaction)
+  end
+
+  subgraph Phase 2
   areSent{7. Have outgoing<br/>messages been<br/>sent?}
   send(8. Send messages)
   updateOutbox(9. Set as sent)
   ack(10. Acknowledge incoming message)
```

Comment:
If splitting the sections, I would assume these numbers would have to be revised to match:
```suggestion
  areSent{1. Have outgoing<br/>messages been<br/>sent?}
  send(2. Send messages)
  updateOutbox(3. Set as sent)
  ack(4. Acknowledge incoming message)
```

Either that, or the second list would have to be kickstarted at 7.

Reply:
Oops, yes. Missed that.